src/System/OS.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Dock\System;
+
+class OS
+{
+    const MAC = 1;
+    const LINUX = 2;
+    const WINDOWS = 3;
+
+    /**
+     * @return int Current Operating System
```

Comment:
Well it returns the "current operating system". Don't think this is necessarily wrong.